cmd/noms/diff/diff_test.go

```diff
@@ -70,7 +72,32 @@ func createStruct(name string, kv ...interface{}) types.Struct {
 	return types.NewStruct(name, fields)
 }
 
-func TestNomsMapdiff(t *testing.T) {
+func pathsFromDiff(v1, v2 types.Value, leftRight bool) []string {
+	dChan := make(chan Difference)
+	sChan := make(chan struct{})
+
+	go func() {
+		Diff(v1, v2, dChan, sChan, leftRight)
+		close(dChan)
+	}()
+
+	paths := []string{}
```

Comment:
done, question, should we always use declarations rather than an empty object initialization. For example, if I have a struct foo, should I use:
var f foo
or
f := foo{}
?